docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>This cell lacks a lot of info in it. Some questions I'd like to have answered:</p><ul><li>If the 1st option is the preferred, why isn't it enabled by default? Why do I need to set the MLRun config param for it (this is not a doc question, but rather an implementation question to Zeev)</li><li>Why would I want to use any of the other methods? </li><li>If using the 2nd method how do I "track the run added to it"? Where do I look for it? This is very vague.</li><li>If using the 3rd method, how do I "look across all experiments for added runs" - is this in MLFlow? What do I do in MLRun after that? Again, vague.</li></ul><p>If 2nd/3rd options are viable, we need to explain them a little more. If we don't want anyone to use them, then no need to put them here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
i am doing the tracking in my code, i guess it's a bit over explaning to the user and might cause some confusion.

Reply:
in the first option we intervein with the run so we know the run id, i can send better explanation for 2 and 3, maybe even remove 3